recipe/meta.yaml

```
@@ -18,7 +18,10 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

```suggestion
    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
    - maturin >=1.0,<2.0                                # [build_platform != target_platform]
```

Pattern matching against other feedstocks:
- https://github.com/conda-forge/py-rattler-feedstock/blob/1067f0c079fedb93b7b30aa69731bca99fdf2612/recipe/meta.yaml#L24
- https://github.com/conda-forge/polars-feedstock/blob/0681f1d6e48beec9f12e2af2de723e3a7d9c1ffd/recipe/meta.yaml#L29